crates/isograph_schema/src/process_type_definition.rs

```diff
@@ -567,13 +568,23 @@ impl UnvalidatedSchema {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq)]
 pub struct FieldMapItem {
     // TODO eventually, we want to support . syntax here, too
+    #[serde(deserialize_with = "deserialize_stringliteral")]
     pub from: StringLiteralValue,
+    #[serde(deserialize_with = "deserialize_stringliteral")]
     pub to: StringLiteralValue,
 }
 
+fn deserialize_stringliteral<'de, D>(deserializer: D) -> Result<StringLiteralValue, D::Error>
+where
+    D: Deserializer<'de>,
+{
```

Comment:
let's not duplicate this

Reply:
Fixed with the latest commit [`bd0d1c6` (#31)](https://github.com/isographlabs/isograph/pull/31/commits/bd0d1c6f9f89a0e3a491c2bc1800a51e30fbfde3)